terraform/aws/analytical-platform-data-production/coat-integration/glue-databases.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_glue_catalog_database" "coat" {
+  name = "coat"
```

Comment:
We already have a database set up as a source for CaDeT in APDP
[cloud_optimisation_and_accountability](https://eu-west-1.console.aws.amazon.com/glue/home?region=eu-west-1#/v2/data-catalog/databases/view/cloud_optimisation_and_accountability?catalogId=593291632749)
so would like further tables added there
This database was created manually - no problem with it moving to TF - assuming other tables in it from other buckets are unaffected.